modules/custom/az_event/az_event.info.yml

```diff
@@ -4,7 +4,8 @@ description: 'Provides Event content type and related configuration.'
 core_version_requirement: '^8.8 || ^9'
 package: 'The University of Arizona'
 dependencies:
-  - az_news
+  - az_core
+  - az_media
   - date_ap_style
```

Comment:
moved everything event used to rely on az_news for to `az_core` and `az_media` so we no longer have the dependency on `az_news`